orchestration/hca_orchestration/resources/load_tag.py

```diff
@@ -34,7 +32,4 @@ def load_tag(init_context: InitResourceContext) -> str:
 
         tag = f"{tag}_{run_id[0:8]}"
 
-    if len(tag) > MAX_LOAD_TAG_LEN:
```

Comment:
In retrospect, contorting our load_tag to play well with Dataflow job name length requirements was a bad idea. This backs that out and just synthesizes a job name unrelated to the load tag.